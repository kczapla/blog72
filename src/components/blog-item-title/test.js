import React from 'react'
import renderer from 'react-test-renderer'

import BlogItemTitle from './index'

describe('discrete text link', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<BlogItemTitle title={'test'}/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
