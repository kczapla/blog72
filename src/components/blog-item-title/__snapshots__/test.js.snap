// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`discrete text link renders correctly 1`] = `
<h2
  className="sc-bdfBwQ kEZLmx"
>
  test
</h2>
`;
